app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -54,12 +58,30 @@ export default function NoteTable({
                   }}
                 />
               </td>
+              <td className="flex flex-row items-center justify-start">
+                <Button style="link" label={note.title} />
+              </td>
+              <td className="truncate text-left" style={{ maxWidth: 100 }}>
```

Comment:
Please refrain from using inline styles. Please use tailwind classes for the same.